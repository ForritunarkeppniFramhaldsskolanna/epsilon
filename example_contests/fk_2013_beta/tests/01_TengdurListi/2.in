29
[0]->NULL
[1]->NULL
[2]->NULL
[3]->NULL
[4]->NULL
[5]->NULL
[6]->NULL
[7]->NULL
[8]->NULL
[9]->NULL
[10]->NULL
[11]->NULL
[15]->NULL
[20]->NULL
[55]->NULL
[99]->NULL
[100]->NULL
[254]->NULL
[9912]->NULL
[100]->[15]->NULL
[15]->[100]->NULL
[1]->[2]->[3]->NULL
[1]->[2]->[3]->[4]->NULL
[4]->[3]->[2]->[1]->NULL
[1]->[2]->[3]->[4]->[5]->NULL
[5]->[4]->[3]->[2]->[1]->NULL
[1823]->[177]->[0]->[1312]->[1212]->[44]->[1374]->[481]->[1231]->[1897]->[999]->[89]->[23]->NULL
[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->NULL
[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->NULL
