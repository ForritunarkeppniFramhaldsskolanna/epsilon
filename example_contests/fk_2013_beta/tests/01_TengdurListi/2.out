[0]->NULL
[1]->NULL
[2]->NULL
[3]->NULL
[4]->NULL
[5]->NULL
[6]->NULL
[7]->NULL
[8]->NULL
[9]->NULL
[10]->NULL
[11]->NULL
[15]->NULL
[20]->NULL
[55]->NULL
[99]->NULL
[100]->NULL
[254]->NULL
[9912]->NULL
[15]->[100]->NULL
[100]->[15]->NULL
[3]->[2]->[1]->NULL
[4]->[3]->[2]->[1]->NULL
[1]->[2]->[3]->[4]->NULL
[5]->[4]->[3]->[2]->[1]->NULL
[1]->[2]->[3]->[4]->[5]->NULL
[23]->[89]->[999]->[1897]->[1231]->[481]->[1374]->[44]->[1212]->[1312]->[0]->[177]->[1823]->NULL
[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->[1]->NULL
[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->[0]->NULL
